### PR TITLE
Changed Header and footer from sections to partials

### PR DESCRIPTION
**Pull request by marcmerce**
Header and Footer are now partials and they use i18n for translation instead of data file

### CODE REVIEW

**Review by EFF — APPROVED** [review]
Seems legit !

Simple question, why did you remove the footer and header content files ? you can't use them in partials ?

File: i18n/en.yaml [hunk 1]
```diff
@@ -15,3 +15,36 @@
...
+  translation: "subscribe"
+
+- id: newsletter
+  translation: "sign up to our newsletter"
+
+- id: newsletter-placeholder
+  translation: "email"
+
+- id: mailchimp
+  translation: "//opfront.us14.list-manage.com/subscribe/post?u=c40666a9e35639e313e69839f&amp;id=657f3802e5"
```
Comment (osylvain) [inline i18n/en.yaml:47]: Having urls in translation files in kind of weird. I understand why you did it though. Leave it like that for now.

**Review by osylvain — APPROVED** [review]
Way to go!

**Comment by marcmerce** [comment]
@EFF After reading the doc for hugo, that's just not a good practice to translate a partial with content files. When you add a partial with i18n to a html file, you write {{ partial "header.html" }}. If you want to use content file, you have to write: 
 
{{ range where .Data.Pages "Section" "header" }}
{{ .Render "header" }}
{{ end }}  

@osylvain How would you do it for the URL? Fr and En are very different URL, I could not find a better way to handle it.. :/

File: layouts/partials/footer.html [hunk 2]
```diff
@@ -1,15 +1,15 @@
 <footer>
     <div class="subfooter newsletter">
         <div id="mc_embed_signup" class="subfooter-content">
-            <span>{{ .Params.newsletter }}</span>
+            <span>{{ i18n "newsletter" }}</span>
             <form action="{{ .Params.mailchimp }}" method="post" id="mc-embedded-subscribe-form"
```
Comment (osylvain) [inline layouts/partials/footer.html]: You sure this is working? You moved the url in the lang file, but you don't seem to use it. I might have missed something, but I thought I'd see something like that instead 
```
{{ i18n "mailchimp" }}
```

**Comment by osylvain** [comment]
@marcmercier1 concerning mailchimp urls:

I would have added params to the toml config file. Something like:
```
[[[Params.MailchimpFr]]] = "//opfront.us14.list-manage........fr url"
[[[Params.MailchimpUrlEn]]] = "//opfront.us14.list-manage.......en url"
```
and then in the markup (I haven't tried, but something like)
```
<form action="{{ $.Param (printf "Mailchimp.%s" .Lang) }}" method="post" ... >
```

Both have pros and cons. Conceptually, it's a config and goes in the config file, but it makes code harder to read. Your decision!

**Pull request merged**